symfony/phpunit-bridge/3.3/bin/phpunit

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env php
+<?php
+
+if (!file_exists(dirname(__DIR__).'/vendor/symfony/phpunit-bridge/bin/simple-phpunit')) {
+    echo "Unable to find the `simple-phpunit` script in `vendor/symfony/phpunit-bridge/bin/`.\n";
+    exit(1);
+}
+if (false === getenv('SYMFONY_PHPUNIT_REMOVE')) {
+    putenv('SYMFONY_PHPUNIT_REMOVE=symfony/yaml');
+}
+if (false === getenv('SYMFONY_PHPUNIT_VERSION')) {
+    putenv('SYMFONY_PHPUNIT_VERSION=6.4');
```

Comment:
-1 for that. We don't want people to have to update the minor version manually all the time. If the PHPUnit version is not set externally, we should keep the default guessed in the bridge IMO (which should be upgraded to use PHPUnit 6 on all supported PHP versions in SF4)

Reply:
Not sure I agree: Flex requires PHP 7.0, so that any 6.* version of phpunit is fine.
The bridge currently set "6.0" for PHP 7.0/7.1 or "6.3" otherwise (>=7.2).
The bridge cannot be updated easily for BC reasons.
The recipe *can* be as often as we want.